cmd/bnsd/client/client.go

```diff
@@ -399,22 +398,20 @@ type CurrenciesResponse struct {
 	Currencies map[string]currency.TokenInfo
 }
 
-func (b *BnsClient) ListCurrencies() (*CurrenciesResponse, error) {
+// GetCurrencies will returns all currencies configured for the blockchain with their token details.
+func (b *BnsClient) GetCurrencies() (CurrenciesResponse, error) {
```

Comment:
Just a minor nit here, if we are talking about idiomatic go, it's never GetSomething, it's just Something, in this case `Currencies()`